app/Http/Middleware/ClearVersion.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class ClearVersion
```

Comment:
Let's move this class to the LaravelHelpers package and rename it to `RemovePathParamMiddleware`